src/ripple/overlay/impl/TrafficCount.h

```diff
@@ -178,50 +178,51 @@ class TrafficCount
         return counts_;
     }
 
+
 protected:
     std::array<TrafficStats, category::unknown + 1> counts_
```

Comment:
A `TrafficCount` object holds many `TrafficStats` objects under a member named `counts_`. This is incredibly confusing naming that we inherited. I would love to see it changed. Here's one idea:

`TrafficCount` becomes `TrafficCounters`, `TrafficStats` becomes `TrafficCounter`, and `counts_` becomes `counters_`. This borrows the "counter" metric name from StatsD, and uses a plural vs. singular distinction to reflect that one is a collection of the other.

Reply:
I think TrafficStats is a more appropriate name given its members. BytesIn, BytesOut, MessagesIn, and MessagesOut. I do agree that TrafficCount should reflect that it contains a collection of metrics.